WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/EventItemViewHolder.kt

```diff
@@ -35,12 +35,12 @@ class EventItemViewHolder(
         summary.text = activity.title
         text.text = activity.description
 
+        ColorUtils.setImageResourceWithTint(
```

Comment:
I moved this outside of the visibility check to ensure that icon is not null when the diffUtil runs in the adapter. The icon used to be hardcoded in the xml file, but now it's created dynamically based on the Backup Feature config enabled flag.